src/main/java/devcource/hihi/boardjpa/service/PostService.java

```diff
@@ -0,0 +1,74 @@
+package devcource.hihi.boardjpa.service;
+
+import devcource.hihi.boardjpa.domain.Post;
+import devcource.hihi.boardjpa.domain.User;
+import devcource.hihi.boardjpa.dto.post.CreatePostDto;
+import devcource.hihi.boardjpa.dto.post.PageCursorDto;
+import devcource.hihi.boardjpa.dto.post.ResponsePostDto;
+import devcource.hihi.boardjpa.dto.post.UpdatePostDto;
+import devcource.hihi.boardjpa.repository.PostRepository;
+import devcource.hihi.boardjpa.repository.UserRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+public class PostService {
+
+    private final PostRepository postRepository;
+    private final UserRepository userRepository;
+
+    public PostService(PostRepository boardRepository, UserRepository userRepository) {
+        this.postRepository = boardRepository;
+        this.userRepository = userRepository;
+    }
+
+    @Transactional
+    public ResponsePostDto createDto(CreatePostDto postDto) {
+        Post post = postDto.toEntity();
+
+        User user = userRepository.findById(postDto.user().getId()).orElseThrow(() -> new RuntimeException("작성자는 null이 될 수 없습니다"));
```

Comment:
```suggestion
        User user = userRepository.findById(postDto.user().getId())
                .orElseThrow(() -> new RuntimeException("작성자는 null이 될 수 없습니다"));
```
p3; 
줄바꿈을 통해 좀 더 깔끔해보일 수 있어요!